doc/api/pyplot_summary.rst

```diff
@@ -1,8 +1,41 @@
-Plotting commands summary
-=========================
+Below we describe several common approaches to plotting with Matplotlib.
+
+.. contents::
+
+The Pyplot API
+--------------
+
+The :mod:`matplotlib.pyplot` module contains functions that allow you to generate
+many kinds of plots quickly. For examples that showcase the use
+of the :mod:`matplotlib.pyplot` module, see the
+:ref:`sphx_glr_tutorials_01_introductory_pyplot.py`
+or the :ref:`pyplots_examples`. We also recommend that you look into
+the object-oriented approach to plotting, described below.
 
 .. currentmodule:: matplotlib.pyplot
 
 .. autofunction:: plotting
 
+The Object-Oriented API
```

Comment:
Should we put this above the Pyplot API section? It could be formatted as an aside/callout/warning if needed. But I think `autofunction` generates a big table of functions (your site appears to be rendering a different PR), so no-one will ever see this recommendation.